File: BOJ/27889.rs
use std::io::*;

fn main() {
    let stdin = read_to_string(stdin()).unwrap();
    let word = stdin.trim();

    let output = match word {
        "NLCS" => "North London Collegiate School",
        "BHA" => "Branksome Hall Asia",
        "KIS" => "Korea International School",
        _ => "St. Johnsbury Academy",
    };

    println!("{output}");
}

File: BOJ/27889.cpp
#include <iostream>
#include <unordered_map>

using namespace std;

int main() {
  ios_base::sync_with_stdio(false);
  cin.tie(nullptr);

  unordered_map<string, string> mp = {
      {"NLCS", "North London Collegiate School"},
      {"BHA",  "Branksome Hall Asia"},
      {"KIS",  "Korea International School"},
      {"SJA",  "St. Johnsbury Academy"}
  };

  string s;
  cin >> s;

  cout << mp[s];

  return 0;
}